### DISCUSSION TITLE
Duplicate parts of the docs - need some sort of include

### DISCUSSION THREAD

**Discussion by audreyfeldroy** [djangopackages/djangopackages#662] [Docs]
These files are almost the same, but diverging due to duplication:

* docs/index.rst and some of README.rst
* docs/contributors.rst and CONTRIBUTORS.txt

Let's see what we can do to avoid the duplication:

* [ ] Look into RST includes
* [x] Look into the Markdown equivalent of RST includes. If it's better, maybe we'll move from RST to Markdown

**Comment by ryancheley** [comment]
The docs were converted from `rst` to `md` in commit 19f0114e0f3868cbbbea0f42b2eecf3954bbf94e
